web/app/components/AnswerView.tsx

```diff
@@ -5,8 +5,7 @@ import { useGameContext } from '~/context/game/GameContext'
 export default function AnswerView(): ReactElement {
     const { state } = useGameContext()
 
-    const onNameClick = () => {
-    }
+    const onNameClick = () => { /* void */}
```

Comment:
Is this function needed?, if not delete

Reply:
Doesn't look like it, i just made the linter happy